app/controllers/followers_controller.rb

```diff
@@ -9,7 +10,6 @@ class FollowersController < ApplicationController
 
   before_action :fetch_follower, only: %i[confirm cancel destroy]
   before_action :set_user_and_custom_domain_config, only: :new
-  before_action :set_body_id_as_app, only: :index
```

Comment:
Had to remove this as otherwise it breaks Inertia - it also uses an `#app` element. Doesn't seem to have an effect anyway.